packages/gatsby-theme/src/components/Contact/ContactItem.js

```diff
@@ -0,0 +1,53 @@
+import PropTypes from "prop-types"
+import { Box } from "theme-ui"
+import Icon from "../Icon"
+import RichText from "../RichText"
+import Heading from "../Heading"
+
```

Comment:
Same as the others, please take this opportunity to add this components to the [component index file](https://github.com/octahedroid/drupal-gatsby-theme/blob/master/packages/gatsby-theme/src/components/index.js) and update this imports
```suggestion
import { Icon } from "ui"
import { RichText } from "ui"
import { Heading } from "ui"
```